js/source/tile_pyramid.js

```diff
@@ -339,5 +339,43 @@ TilePyramid.prototype = {
                 };
             }
         }
+    },
+
+    /**
+     * Search through our current tiles and attempt to find the tiles that
+     * cover the given bounds.
+     * @param {Array<Coordinate>} bounds - opposite corners of bounding rect.
+     * @returns {Array<Object>} result items have {tile, minX, maxX, minY, maxY}, where min/max bounding values are the given bounds transformed in into the coordinate space of this tile.
+     * @private
+     */
+    tilesIn: function(bounds) {
+        var result = [];
+        var ids = this.orderedIDs();
```

Comment:
Would it make sense to use `renderedIDs()` instead of `orderedIDs()` here?

Also: I figured the `bounds` parameter could be optional, and if omitted, this method would serve up all the tiles in view.  Anything wrong with just using all of the  `renderedIDs()`?